lib/plugins/entities.js

```diff
@@ -271,7 +270,7 @@ function inject (bot, { version }) {
 
   // TEMP: Misnamed packet in 1.17 protocol, entityIds is a single varint
```

Comment:
in 1.17 it's now a single varint named entityId

Reply:
re this: before this is merged, we should just emit a RemoveEntities packet with the one entity from the RemoveEntity packet because in 1.17.1, we get the RemoveEntities packet back